packages/datadog-instrumentations/src/next.js

```diff
@@ -38,6 +38,27 @@ function wrapHandleApiRequest (handleApiRequest) {
   }
 }
 
+// next 13.2 handleApiRequest uses a different set of parameters
+function wrapHandleApiRequestWithMatch (handleApiRequest) {
+  return function (req, res, query, match) {
+    return instrument(req, res, () => {
+      const promise = handleApiRequest.apply(this, arguments)
+
+      return promise.then(handled => {
+        if (!handled) return handled
+
+        const page = (typeof match === 'object' && typeof match.definition === 'object')
+          ? match.definition.page
```

Comment:
From [this other PR](https://github.com/DataDog/dd-trace-js/pull/2911/files#diff-2459f26b869c7f2688c130857769922a2b367bf5e9d39602c91e60c4584a4047R44-R46) it sounds like `pathname` could be better than `page` here. Did you consider both options, and if yes, what makes `page` better?

Reply:
I did not personally encounter any differences between the two in our applications usage, However after looking further at next-server's implementation, i updated it to pathname which is the what handleApiRequest ->     return this.runApi(req, res, query, params, pathname, filename) uses/passes along.

We dont typically localize api routes and I believe this is specific to /pages/api/*